web/annotation_layer_builder.css

```
@@ -15,6 +15,10 @@
 
 :root {
   --annotation-unfocused-field-background: url("data:image/svg+xml;charset=UTF-8,<svg width='1px' height='1px' xmlns='http://www.w3.org/2000/svg'><rect width='100%' height='100%' style='fill:rgba(0, 54, 255, 0.13);'/></svg>");
+  --inner-focus-line-color: Highlight;
```

Given how/where this is being used below, I'd really expect this variable to be named as follows instead:
```suggestion
  --input-focus-border-color: Highlight;
```